engine/src/cmd/planet_generic.cpp

```diff
@@ -348,6 +364,8 @@ Unit* Planet::beginElement( QVector x,
                                                                QVector( 0, 0, 0 ), this, ourmat, ligh, faction, fullname, inside_out ) );
             un = p;
             p->SetOwner( this );
+            cout << "Created planet " << fullname << " of type " << p->fullname << " orbiting " << this->fullname << endl;
```

Comment:
Would it be too much to ask that this PR use `BOOST_LOG_TRIVIAL` instead of `cout` output directly?

Reply:
It would certainly not be too much :)

Reply:
Looks like we still have a few `cout`s here and there.